saw-core/src/Verifier/SAW/Term/Pretty.hs

```diff
@@ -114,28 +114,19 @@ depthAllowed _ _ = True
 
 -- | Precedence levels, each of which corresponds to a parsing nonterminal
 data Prec
-  = PrecNone   -- ^ Nonterminal 'Term'
+  = PrecNone   -- ^ Nonterminal 'Term' or "sepBy(Term, ',')"
+  | PrecTerm   -- ^ Nonterminal 'Term'
```

Comment:
Both `PrecTerm` and `PrecNone` have the description "Nonterminal `Term`", but is there ever a situation where you'd use `PrecNone` for something besides a tuple?

Reply:
Now that I think about it, no, actually there's not. In that case it's probably a good idea to rename the constructor to something like `PrecCommas`, to reflect its actual purpose.